sql/src/test/java/io/crate/metadata/TableIdentTest.java

```diff
@@ -66,4 +66,12 @@ public void testFQN() throws Exception {
         ti = new TableIdent("s", "t");
         assertThat(ti.fqn(), is("s.t"));
     }
+
+    @Test
+    public void testFqnFromIndexName() throws Exception {
```

Comment:
Please also add a test for the `IllegalArgumentException` case(s) - index name with 3 or more parts